CMakePresets.json

```diff
@@ -38,14 +39,26 @@
     },
     {
       "name": "release",
+      "inherits": ["default"],
       "displayName": "Release build",
       "description": "Release build with Ninja generator",
       "generator": "Ninja",
-      "binaryDir": "${sourceDir}/build-release",
       "cacheVariables": {
         "CMAKE_BUILD_TYPE": "Release",
-        "CMAKE_COMPILE_WARNING_AS_ERROR": "OFF",
-        "CMAKE_INSTALL_PREFIX": "/opt/ats",
+        "CMAKE_COMPILE_WARNING_AS_ERROR": "ON",
+        "BUILD_EXPERIMENTAL_PLUGINS": "ON"
```

Comment:
This changes the behavior of the release build configuration. I find the change to enable -Werror particularly intriguing; was the previous behavior incorrect? Why is this change done here?

Reply:
Oh, I tried to homogenize these but maybe they weren't supposed to be the same? idk, I'll clean it up.